src/modules/Lan.rb

```diff
@@ -348,38 +348,47 @@ def Read(cache)
       return false if Abort()
       # Progress step 4/8
       ProgressNextStage(_("Reading network configuration...")) if @gui
-      NetworkConfig.Read
+      begin
```

Comment:
I wonder why only a part of this long method is worth rescuing. Split off a method with a telling name? Rescue the whole method?

Reply:
```Lan::Read``` mixes GUI preparation, GUI handling and configuration reading all together. I wanted to catch only exceptions related to configuration reading.